client/src/components/Draftail/CommentableEditor/CommentableEditor.tsx

```diff
@@ -291,6 +291,39 @@ function updateCommentPositions({ editorState, comments, commentApp }:
   });
 }
 
+export function findLeastCommonCommentId(block: ContentBlock, offset: number) {
```

Comment:
I think we could do with a comment here to describe what this function is meant to do

Reply:
Good point, thanks!